Packs/PhishUp/Playbooks/PhishUp_Mail_Scanner.yml

```diff
@@ -0,0 +1,790 @@
+            value:
+              simple: inputs.ShouldPhishUpActionWork
+            iscontext: true
+          right:
+            value:
+              simple: "False"
+    view: |-
+      {
+        "position": {
+          "x": -430,
+          "y": 1010
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: 7076e7be-93c3-4fd2-8709-1e4ee1f773e0
+    type: regular
+    task:
+      id: 7076e7be-93c3-4fd2-8709-1e4ee1f773e0
+      version: -1
+      name: Get PhishUp Action From Instance
+      description: Get chosen PhishUp action option from instance
+      script: '|||phishup-get-chosen-action'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "17"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -720,
+          "y": 1510
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: dd209e4d-1533-47e6-8e63-d12ed2ec9abf
+    type: condition
+    task:
+      id: dd209e4d-1533-47e6-8e63-d12ed2ec9abf
+      version: -1
+      name: Execute PhishUp Action
+      description: Execute PhishUp Action
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "19"
+      Delete:
+      - "9"
+      Move to Spam:
+      - "18"
+    separatecontext: false
+    conditions:
+    - label: Delete
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: PhishUp.Action
+            iscontext: true
+          right:
+            value:
+              simple: Delete Mail
+    - label: Move to Spam
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: PhishUp.Action
+            iscontext: true
+          right:
+            value:
+              simple: Move to SPAM
+    view: |-
+      {
+        "position": {
+          "x": -730,
+          "y": 1670
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: 59a07f93-1155-4a7f-8fe0-b7e041c32c43
+    type: regular
+    task:
+      id: 59a07f93-1155-4a7f-8fe0-b7e041c32c43
+      version: -1
+      name: Move Mail to Spam Folder
+      description: Moves an email to a different folder.
+      script: '|||gmail-move-mail'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      add-labels:
+        simple: SPAM
+      message-id:
+        simple: ${incident.emailmessageid}
```

Comment:
I believe this should be a playbook input and not fetched directly from the incident since in some scenarios, this playbook may be used as a sub playbook.
So, a playbook input like `GmailMessageID` can be configured and then be used here.